tests/kafkatest/tests/streams/streams_eos_test.py

```diff
@@ -166,7 +166,6 @@ def abort_streams(self, keep_alive_processor1, keep_alive_processor2, processor_
 
     def wait_for_startup(self, monitor, processor):
         self.wait_for(monitor, processor, "StateChange: REBALANCING -> RUNNING")
-        self.wait_for(monitor, processor, "processed [0-9]* records from topic")
```

Comment:
Is this verification strictly needed? If the Streams client has only standby tasks assigned, it will not process records but it will be in state RUNNING. A Streams client that has only standby tasks assigned seems a valid situation to me that should not make the test fail.

Reply:
Does this test even use standbys?

IIRC, the idea was to ensure that the instance really started to process data, so we don't move to the next step, too quickly in the test.

Reply:
According to the logs the test uses even 2 standby replicas (`num.standby.replicas = 2`). The git history says it was configured that way long time ago on June 8, 2017 by a guy named @mjsax.  🙂  

And even if it wouldn't, there is still the possibility that a warm-up standby task is assigned to the client, before the actual active task is moved to it. Note `with followup probing rebalance` in the logs I posted. We could wait for settlement of the assignment, but that would extend the runtime of the test. I get your point about ensuring to not move to the next step too quickly, though.